tests/CMakeLists.txt

```diff
@@ -5,36 +5,42 @@
 # at once with CMake's ctest command line tool from the build folder.
 enable_testing()
 
+################################################################################
+# General tests and tools
+
 add_executable(aviftest aviftest.c)
 if(AVIF_LOCAL_LIBGAV1)
     set_target_properties(aviftest PROPERTIES LINKER_LANGUAGE "CXX")
 endif()
 target_link_libraries(aviftest avif ${AVIF_PLATFORM_LIBRARIES})
 add_test(NAME aviftest COMMAND aviftest ${CMAKE_CURRENT_SOURCE_DIR}/data)
 
-add_executable(avifincrtest avifincrtest.c avifincrtest_helpers.c)
+add_executable(avifyuv avifyuv.c)
+if(AVIF_LOCAL_LIBGAV1)
+    set_target_properties(avifyuv PROPERTIES LINKER_LANGUAGE "CXX")
+endif()
+target_link_libraries(avifyuv avif ${AVIF_PLATFORM_LIBRARIES})
+foreach(AVIFYUV_MODE limited rgb) # Modes drift and premultiply take more than 2 minutes each so they are disabled.
+    add_test(NAME avifyuv_${AVIFYUV_MODE} COMMAND avifyuv -m ${AVIFYUV_MODE})
+endforeach()
+
```

Comment:
Optional: The `if(AVIF_ENABLE_COVERAGE)` part at lines 78-91 probably could be moved to this section because it seems to depend on `aviftest` only.